validation/jinja2/validation/segment_details.html

```diff
@@ -1,8 +1,53 @@
+<script>
+    function showDiv(name, suggestions=null) {
+        document.getElementById('edit').style.display = "block";
+        document.getElementById('id_cree').focus()
+        analysis = '';
+
+        if (suggestions == null) {
+            translation = document.getElementById(name + '-translation').innerHTML;
+        }
+
+        else {
```

Comment:
Haha, maybe we should figure out what JavaScript coding style guide to use in this project!

Reply:
I’ve been quite happy with `prettier` for my JS work. I grumbled a bit when they changed the default style—and the style for all my existing code—when going from 1.x to 2.0, but the one-time change turned out not to be an issue at all.